packages/web/src/pages/explore-page/components/desktop/ExploreSection.tsx

```diff
@@ -126,12 +131,53 @@ export const ExploreSection: React.FC<ExploreSectionProps> = ({
             css={{ minWidth: 'max-content', overflow: 'visible' }}
             pv='2xs'
           >
-            {data
-              ? data?.map((id) => <Card key={id} id={id} size='s' />)
-              : Array.from({ length: 6 }).map((_, i) => (
-                  // loading skeletons
-                  <Card key={i} id={0} size='s' />
-                ))}
+            {Tile
```

Comment:
this seems a bit funky, maybe refactor this to be slightly more legible? probably dont to inline function calls?